src/Native/Unix/System.Native/pal_io.c

```diff
@@ -49,6 +49,25 @@ extern int     getpeereid(int, uid_t *__restrict__, gid_t *__restrict__);
 extern ssize_t  getline(char **, size_t *, FILE *);
 #endif
 
+#if defined(__linux__)
+#if defined(__x86_64__)
+#define PAL_COPY_FILE_RANGE_SYSCALL 326
+#elif defined(__i386__)
+#define PAL_COPY_FILE_RANGE_SYSCALL 377
+#elif defined(__arm__)
+#define PAL_COPY_FILE_RANGE_SYSCALL 391
+#elif defined(__aarch64__)
+#define PAL_COPY_FILE_RANGE_SYSCALL 285
+#else
+#    include <syscall.h>
+#    if defined(__NR_copy_file_range)
+#        define PAL_COPY_FILE_RANGE_SYSCALL __NR_copy_file_range
+#    else
+#        warning copy_file_range() unavailable
```

Comment:
What is the purpose of emitting a warning? If we are supposed to fallback to a less efficient copy method when copy_file_range does not exist in the system, why would the user care about getting warned about this?

Reply:
We might want to add the syscall to the hardcoded `#define` list; without a warning, it's probably not going to be noticed.  (No problems in removing the warning, though; it's not like support for new arches is added every day, and it's very likely you're going to grep for other architecture names anyway to see if you need to do something different for the new arch.)

Reply:
Let's remove the warning.